test/ValidationFramework/Validation.hpp

```diff
@@ -0,0 +1,125 @@
+/// Apache License 2.0
+
+#include <iostream>
+
+#include <Global.test.hpp>
+
+class Validation : public ::testing::Test
+{
+   public:
+    enum class OutputQuantity
```

Comment:
Will eventually need some static function that clusters a `String` into an `OutputQuantity`

Reply:
Yep, same comment on my part below ✅

Reply:
Yup